KOGITO-7271 Add swf editor(tooling) docs (#33)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc

= Chrome Extension for Kogito Serverless Workflow Editor

The Chrome Extension for the Kogito Serverless Workflow Editor allows you to view and edit Serverless Workflow files in GitHub.

== Installing Chrome the Extension
The chrome extension can be installed in two ways.

* You can visit chrome web store with link:{kogito_swf_editor_chrome_webstore_url}[Chrome extension for Kogito Serverless Workflow Editor] and click on *Add to Chrome* button.

.Chrome Extension Page
image::tooling/serverless-workflow-editor/swf-editor-chrome-extension-page.png[]

* You can download and extract the 'chrome_extension_serverless_workflow_editor_VERSION.zip' file from the link:{kie_tools_releases_page_url}[kie-tools] releases page in GitHub, and then in the upper-right corner in Chrome, go to *Customize and control* → *Settings* → *Extensions* → *Load unpacked* and open the extracted dist folder.

== Working with Chrome Extension

> Once you are done with the installation, create a Serverless Workflow file(ending with *.sw.json* or *.sw.yaml*) and push it to GitHub or visit any GitHub page with a similar type of file and click on it to open the editor.

Reviewer: ```suggestion
Once you are done with the installation, create a Serverless Workflow file (ending with *.sw.json* or *.sw.yaml*) and push it to GitHub or visit any GitHub page with a similar type of file and click on it to open the editor.
```

= Chrome Extension for Serverless Workflow Editor

> The Chrome Extension for Serverless Workflow Editor allows you to view and edit Serverless Workflow files in github.

Reviewer 2: ```suggestion
The Chrome Extension for Serverless Workflow Editor allows you to view and edit Serverless Workflow files in GitHub.
```

== Installing Chrome Extension
The chrome extension can be installed in two ways.

> * You can visit chrome web store with https://chrome.google.com/webstore/detail/serverless-workflow-edito/ijamhkegfogkfnmfnfkjdiadomlphfej[Chrome extension for serverless workflow editor] and click on *Add to Chrome* button.

Reviewer 2: maybe use the link patter as we are doing on other guides?
common link on antora.yaml and links only used here, below the header.

.Chrome Extension Page
image::tooling/serverless-workflow-editor/swf-editor-chrome-extension-page.png[]

> * You can download and extract the chrome_extension_serverless_workflow_editor_VERSION.zip file from the https://github.com/kiegroup/kie-tools/releases[kie-tools] releases page in GitHub, and then in the upper-right corner in Chrome, go to *Customize and control* → *Settings* → *Extensions* → *Load unpacked* and open the extracted dist folder.

Reviewer 2: ```suggestion
* You can download and extract the 'chrome_extension_serverless_workflow_editor_VERSION.zip' file from the https://github.com/kiegroup/kie-tools/releases[kie-tools] releases page in GitHub, and then in the upper-right corner in Chrome, go to *Customize and control* → *Settings* → *Extensions* → *Load unpacked* and open the extracted dist folder.
```

.Chrome Extension zip release page
image::tooling/serverless-workflow-editor/swf-editor-chrome-extension-release-page.png[]

== Working with Chrome Extension
Once you are done with the installation, create a Serverless Workflow file(ending with *.sw.json* or *.sw.yaml*) and push it to github or visit any github page with a similar type of file and click on it to open the editor.

> .Serverless Workflow file opened in github

Reviewer 2: ```suggestion
.Serverless Workflow file opened in GitHub
```

image::tooling/serverless-workflow-editor/swf-editor-in-github.png[]

The editor initially loads up in the *read-only* mode. To change it to *edit* mode, you need to click on the pencil icon on the top-right corner.

> You can switch to *full-screen* mode to switch to a full screen editor. You can hide the editor and view the source code on github using the *See as source* button.

Reviewer 2: ```suggestion
You can switch to *full-screen* mode to switch to a full screen editor. You can hide the editor and view the source code on GitHub using the *See as source* button.
```
